net/cryfs/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                1.0
+PortGroup                 github 1.0
+PortGroup                 cmake 1.1
+PortGroup                 active_variants 1.1
+
+
+github.setup              cryfs cryfs 0.10.2
+github.tarball_from       releases
+
+license                   LGPL-3
+
+categories                net fuse
+platforms                 darwin
+
+maintainers               {@jolan78 openmaintainer}
+
+description               Cryptographic filesystem for the cloud
+long_description          CryFS encrypts your files, so you can safely store them anywhere.\
+                          It works well together with cloud services like Dropbox, iCloud,\
+                          OneDrive and others
+
+homepage                  https://www.cryfs.org/
+
+checksums                 rmd160  9abdb5e59263cf8ea978470f4c6a4086ddd8a860\
+                          sha256  004c7fdf537062e197aaa0bcdafd2682904a279802441bb6063cb72c1fba52d2\
+                          size    10187912
+
+extract.mkdir             yes
+
+depends_build             port:openssl\
+                          port:python27\
```

Comment:
Does this package run python code? Generally, at this point, we would want a port to depend on python3 of some flavor as python27 goes out of support in less than six weeks, and I don't think this is the correct way to express the dependency. @reneeotten would know the right thing.

Reply:
No, it is only required at build time to get the git version. I will change this to python36 if that looks good for you.
It is also required for building a test suite which is not built by this port.